unittests/AST/ASTImporterTest.cpp

```diff
@@ -2096,7 +2096,6 @@ TEST_P(ImportFunctions, ImportImplicitFunctionsInLambda) {
       FromTU, functionDecl(hasName("foo")));
   auto *ToD = Import(FromD, Lang_CXX);
   EXPECT_TRUE(ToD);
-  Decl *ToTU = ToAST->getASTContext().getTranslationUnitDecl();
```

Comment:
If this is fixed the another warning can be fixed too, there should be a warning related to `;` character?

Reply:
I didn't get any other warning.